SafeguardSessions/test/Integration/TestAuthentication.query.pq

```diff
@@ -94,8 +94,8 @@ TestAuthenticateRaisesError = () =>
             in
                 facts,
         cases = {
-            {401, "Authentication Error", "Something went wrong while authenticating"},
-            {403, "Authorization Error", "Not authorized to access the given resource"},
+            {401, "Authentication Error", "The username or password is invalid"},
+            {403, "Authorization Error", "You are not authorized to access the given resource"},

```

Comment:
In this test `CustomErrors.AuthenticationError` and `CustomErrors.AuthorizationError` is raised when error is mocked in https://github.com/OneIdentity/SafeguardPowerBI/blob/ce2b0a8e564e8eb69df8df8c412612d27cd7c6a8/SafeguardSessions/SafeguardSessions.pq#L92
it's not clear to me if this is because `ResponseHandler` should handle the error, but for some reason it isn't or if this change just is not needed.

Reply:
My bad. Please ignore me. I had a left over file contaminating my build.